docs/002-basics-of-ci-with-actions.md

```diff
@@ -48,13 +47,13 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x]
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
Node Version 18 is EOL, and 20 is also almost gone. So while we update, why not just update to the supported version 22 and 24? 🙂 

```suggestion
        node-version: [22.x, 24.x]
```